### PR TITLE
split apt beta repository to separate endpoint

### PR DESCRIPTION
**Pull request by blockloop**

### CODE REVIEW

File: scripts/deploy.sh [hunk 1]
```diff
@@ -125,13 +134,21 @@ function force_release_enabled() {
 	return 1
 }
 
+function cleanup() {
+	if is_enabled "${SKIP_CLEANUP}" ; then
+		anounce "SKIP_CLEANUP is set to ${SKIP_CLEANUP}, skipping this step"
```
Comment (bsnyder788) [inline scripts/deploy.sh]: anounce? spelling? don't think I've seen that command before either way

Comment (blockloop) [reply scripts/deploy.sh]: That's an existing function in the script

Comment (blockloop) [reply scripts/deploy.sh]: `s/anounce/announce/g`

Comment (blockloop) [reply scripts/deploy.sh]: Pushed a spelling fix

**Pull request merged**